### PR TITLE
Update function to add info if successfully or not

### PR DESCRIPTION
**Pull request by Transformer-cloud**

### CODE REVIEW

File: azure/functionCode/agent_image_embeddings_calculator/function_app.py [hunk 1]
```diff
@@ -18,11 +18,11 @@
 
 
 @app.function_name(name="imageHandler")
-@app.blob_trigger(arg_name="imageBlob", path=blob_path,
+@app.blob_trigger(arg_name="blob", path=blob_path,
```
Comment (itayamir) [inline azure/functionCode/agent_image_embeddings_calculator/function_app.py:21]: what's this change?

File: azure/functionCode/agent_image_embeddings_calculator/function_app.py [hunk 2]
```diff
@@ -18,11 +18,11 @@
 
 
 @app.function_name(name="imageHandler")
-@app.blob_trigger(arg_name="imageBlob", path=blob_path,
+@app.blob_trigger(arg_name="blob", path=blob_path,
                   connection="imageHandler")
-def agent_image_embeddings_calculator(imageBlob: func.InputStream):
+def agent_image_embeddings_calculator(blob: func.InputStream):
```
Comment (itayamir) [inline azure/functionCode/agent_image_embeddings_calculator/function_app.py:23]: And this?

File: azure/functionCode/agent_image_embeddings_calculator/function_app.py (hunk 1 above)
Comment (Transformer-cloud) [reply azure/functionCode/agent_image_embeddings_calculator/function_app.py:21]: python variable need to be lower case

File: azure/functionCode/agent_image_embeddings_calculator/function_app.py (hunk 2 above)
Comment (Transformer-cloud) [reply azure/functionCode/agent_image_embeddings_calculator/function_app.py:23]: python variable need to be lower case

**Pull request merged**